src/components/DpRadio/DpRadioTs.vue

```diff
@@ -0,0 +1,117 @@
+<template>
```

Comment:
Why do you duplicate the component?

Reply:
And: Do we want to Postfix the Ts-Components?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/291/commits/9997286261f720374fb6e9eb0b47b1dff1469140

I initially wanted to make two PR's, one for the UI, and the other for the Core, and after test of component with typescript, can I return name of component.
But also we can test TS component with same name, before Merge.